crates/bevy_winit/src/lib.rs

```diff
@@ -37,6 +41,8 @@ pub struct WinitPlugin;
 impl Plugin for WinitPlugin {
     fn build(&self, app: &mut App) {
         app.init_non_send_resource::<WinitWindows>()
+            .init_resource::<WinitPersistentState>()
```

Comment:
Is this needed as a resource? It seems it's never used outside of the event loop, couldn't it be a captured variable there?

Reply:
End users may want to configure this at run-time, so I like it as a resource.

Reply:
This struct is only an internal state, it should not be touched by end users. And it's private, so not usable anyway

Reply:
What is the proper way to persist state here without a resource?